js/renderer.ts

```diff
@@ -1129,26 +1149,26 @@ export abstract class ReactUiBase extends Component implements ReactComp {
     };
 
     onPointerClick(e: PointerEvent) {
-        const x = e.clientX;
-        const y = e.clientY;
+        const x = e.clientX * this.pixelSizeAdjustment * this.dpr;
+        const y = e.clientY * this.pixelSizeAdjustment * this.dpr;
         this.onClick({x, y, e});
     }
 
     /** 'pointerdown' event listener */
     onPointerDown(e: PointerEvent): NodeWrapper | null {
         /* Don't care about secondary pointers or non-left clicks */
         if (!e.isPrimary || e.button !== 0) return null;
-        const x = e.clientX;
-        const y = e.clientY;
+        const x = e.clientX * this.pixelSizeAdjustment * this.dpr;
```

Comment:
It might makes sense to extract this into a `this.dpiAdjust(e.clientX)`

Reply:
I agree. It crossed my mind too, the repeating calculation should be extracted in a function.